public/data/job-openings-data.json

```diff
@@ -231,6 +231,54 @@
           "experience": "-"
         }
       ]
+    },
+    {
+      "id": 7,
+      "companyName": "iTechOps Cloud Pvt Ltd",
+      "location": "Ahmedabad",
+      "bio": "iTechOps helps customers to turn creative idea's into digital softwares. iTechOps provides satisfactory, high-quality, cost-effective DevOps and Full Stack Development Consulting services. Join our growing team to learn cutting-edge tools & technologies.",
+      "companyLogoUrl": "https://itechops.com/assets/ilogo_128.jpg",
+      "companyWebsite": "https://itechops.com/",
+      "companyCareerPage": "https://itechops.com/career",
```

Comment:
Page is giving 500, please check

Reply:
@NishargShah, sorry for the inconvenience. I have fixed the issue. Please check and approve.